docs/cypress-testing-library/intro.md

```diff
@@ -53,9 +52,12 @@ To show some simple examples (from
 
 ```javascript
 cy.findAllByText('Jackie Chan').click()
-cy.queryByText('Button Text').should('exist')
-cy.queryByText('Non-existing Button Text').should('not.exist')
-cy.queryByLabelText('Label text', {timeout: 7000}).should('exist')
+cy.findByText('Button Text').should('exist')
+cy.findByText('Non-existing Button Text').should('not.exist')
+cy.findByLabelText('Label text', {timeout: 7000}).should('exist')
```

Comment:
`find*` should be preferred over `query*`

Reply:
Agreed. I'm glad that we're keeping all of the `query*` stuff off here considering we're going to be removing it. Thanks!